tests/README.md

```diff
@@ -36,7 +36,7 @@ SWSS, Redis, and all the other required components run inside a virtual switch D
     ```
     sudo modprobe team
     sudo apt install python3-pip net-tools ethtool vlan libnl-nf-3-200 libnl-cli-3-200
-    sudo pip3 install docker pytest flaky redis distro
+    sudo pip3 install docker pytest flaky redis distro dataclasses fstring
```

Comment:
Can you check what version of `python3` you have? I wonder if this is some weird `python3.7` vs. `python3.5` issue, b/c I believe these are both available by default now.

Reply:
I have Python 3.6.9.

Reply:
Ubuntu 18.04.3 LTS

Reply:
Ok, that is why then! Let's roll with this for now b/c it doesn't seem to hurt python3.8, but let's update the instructions to be more explicit about which version of python we're using later.